src/app.py

```diff
@@ -3,24 +3,35 @@
 import pandas as pd
 from dash.dependencies import Input, Output
 
-# Sample data for pie chart
-data = pd.DataFrame({
-    'Location': ['Downtown', 'Hastings-Sunrise', 'Kitsilano', 'Marpole', 'Grandview-Woodland', 'Other'],
-    'Issues': [100, 150, 200, 250, 300, 350]
-})
-
-# Load data for bar chart
-df = pd.read_csv('data/clean/rentals_with_property_value.csv')
-
-def create_pie_chart(data):
-    chart = alt.Chart(data, title='Number of Rental Issues').mark_arc().encode(
-        theta=alt.Theta('Issues', type='quantitative'),
-        color=alt.Color('Location', type='nominal'),
-        tooltip=['Location', 'Issues']
-    ).properties(
-        width=400,
-        height=400
+property_values = pd.read_csv('data/clean/rentals_with_property_value.csv')
+issues = pd.read_csv('data/clean/rental_issues_clean.csv')
+
+
+def create_pie_chart(data, selected_region):
+
+    base = alt.Chart(data, title='Number of Rental Issues').mark_arc().encode(
+        theta=alt.Theta('total_outstanding', type='quantitative'),
+        tooltip=['geo_local_area', 'total_outstanding']
```

Comment:
II think if we add an extra line to specify order here we can sort the pie chart values by their size, it might help with visualization and looking for neighbourhoods with the highest # of issues: 
```suggestion
        tooltip=['geo_local_area', 'total_outstanding'],
        order=alt.Order('total_outstanding:Q', sort='descending')
```

Reply:
That's a good idea! just fixed it